generators/upgrade/upgrade.spec.ts

```diff
@@ -86,13 +86,13 @@ Initial version of upgradeTest generated by generator-jhipster@undefined"
     });
 
     it('generated git commits to match snapshot', () => {
-      const commits = execaCommandSync('git log --pretty=format:%s', { stdio: 'pipe', reject: false }).stdout;
+      const commits = execaSync('git log --pretty=format:%s', { stdio: 'pipe', reject: false }).stdout;
```

Comment:
I don’t think this is correct.

Reply:
I'll try to find the right command, because execcaCommandSync has been deprecated https://github.com/sindresorhus/execa/issues/1055

Reply:
rolled back: will take a look later because it still works...